vars/grafanaAgent.groovy

```diff
@@ -0,0 +1,46 @@
+boolean supported() {
+ //   int exitCode = (sh (returnStatus: true, script: 'systemctl list-units --all | grep grafana-agent')) as int
+    int exitCode = 0
+    return exitCode == 0
+}
+
+void writeEnvVars(String file, Map<String, String> envVars=[:]) {
+    envVars.each {k, v -> 
+        sh '''sudo sed -i 's/''' + k + '''=.*//' /etc/default/grafana-agent'''
+        echo '''echo "''' + k + '''=''' + v + '''" | sudo tee -a /etc/default/grafana-agent'''
```

Comment:
Yess we are using 

```
echo "ENV_VAR=value" | sudo tee -a ....
```


We cannot use `echo "...." > ....` because we need sudo permissions for it